python/python/tests/tests_server_modules/search/test_ft_dropindex.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+from typing import List
+
+import pytest
+from glide.async_commands.server_modules import ft
+from glide.async_commands.server_modules.ft_options.ft_create_options import (
+    DataType,
+    Field,
+    FtCreateOptions,
+    TextField,
+)
+from glide.config import ProtocolVersion
+from glide.constants import OK
+from glide.exceptions import RequestError
+from glide.glide_client import GlideClusterClient
+
+
+@pytest.mark.asyncio
+class TestFtDropIndex:
+    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_ft_dropindex(self, glide_client: GlideClusterClient):
+        # Index name for the index to be dropped.
+        indexName = str(uuid.uuid4())
```

Comment:
mayeb add one byte parameter to test as well?

Reply:
As discussed we will add byte tests after all commands are added.